src/components/explore/AggregationsOverTimeGraph.svelte

```diff
@@ -144,17 +144,25 @@
     $showContextMenu = true; // eslint-disable-line no-unused-vars
   }
 
+  const linearMetrics = [
+    'histogram-linear',
+    'quantity',
+    'counter',
+    'labeled_counter',
+  ];
```

Comment:
These metrics have linear graph type, but initially I only had `histogram-linear` and `quantity` on this list, so other linear graphs such as `counter` and `labeled_counter` did not scale (see [counter](https://dev.glam.nonprod.dataops.mozgcp.net/fenix/probe/events_total_uri_count/explore?ping_type=*) and [labeled_counter](https://dev.glam.nonprod.dataops.mozgcp.net/fenix/probe/logins_store_unlock_error_count/explore?ping_type=*&timeHorizon=ALL))

After adding them to the list, we can rescale y-axis for `counter` and `labeled_counter` now. 

Working example of a `counter` metric:
![2021-06-22 12 28 21](https://user-images.githubusercontent.com/28797553/122987371-6085e280-d355-11eb-9cbc-633e1ed1557b.gif)